packages/graphql/schemas/schema.graphql

```diff
@@ -742,7 +742,7 @@ type Mutation {
   cloudProjectCreate(ciProviders: [String!], name: String!, orgId: ID!, public: Boolean!): CloudProject
 
   """Request access to an organization from a projectId"""
-  cloudProjectRequestAccess(projectSlug: String!): Boolean
```

Comment:
It looks like this branch might be out of date w/ 10.0-release, looks like it already has this change: https://github.com/cypress-io/cypress/blob/567034459089d9d53dfab5556cb9369fb335c3db/packages/graphql/schemas/schema.graphql#L910-L911